### PR TITLE
ELPP-3163 proofreader 0.2

### PR DESCRIPTION
**Pull request by giorgiosironi**
Built using https://github.com/elifesciences/builder-base-formula/pull/136 and will fail until that is available.

### CODE REVIEW

File: .php_cs [hunk 1]
```diff
@@ -1,12 +1,17 @@
 <?php
 
-$finder = Symfony\CS\Finder\DefaultFinder::create()
-    ->exclude('pattern-library')
+$finder = PhpCsFixer\Finder::create()
     ->name('update')
+    ->exclude('var')
```
Comment (thewilkybarkid) [inline .php_cs:5]: Looks like we should ignore `cache`, `src/eLife/Medium/Model/Map` and `src/eLife/Medium/Model/Base` too.

Comment (giorgiosironi) [reply .php_cs:5]: I don't have these folders locally, and `cache` is in `var/cache/` (if it's that one). Something needs to run for them to be created?

Comment (thewilkybarkid) [reply .php_cs:5]: `cache` exists on `prod` (and `.gitignore`), but looks like it was moved to `var/cache`. The others are created by Propel.

Comment (giorgiosironi) [reply .php_cs:5]: It looks like they are in `var/cache/propel/src/eLife/Medium/Model` now?

Comment (thewilkybarkid) [reply .php_cs:5]: 💡

**Comment by giorgiosironi** [comment]
Changed the pillar to 0.2, now green (and will block all other medium pull requests until merged).

**Pull request merged**